public/pages/Emails/CreateRecipientGroup.tsx

```diff
@@ -66,19 +66,25 @@ export function CreateRecipientGroup(props: CreateRecipientGroupProps) {
   };
 
   useEffect(() => {
-    setBreadcrumbs([
+    const { edit } = props;
+    const breadcrumbs = [
       BREADCRUMBS.NOTIFICATIONS,
       BREADCRUMBS.EMAIL_GROUPS,
-      props.edit
-        ? BREADCRUMBS.EDIT_RECIPIENT_GROUP
-        : BREADCRUMBS.CREATE_RECIPIENT_GROUP,
-    ]);
-    window.scrollTo(0, 0);
+    ];
+    if (edit) {
+      if (getUseUpdatedUx()) {
+        breadcrumbs.push(BREADCRUMBS.EDIT_RECIPIENT_GROUP_DETAILS(name))
+      }
+      breadcrumbs.push(BREADCRUMBS.EDIT_RECIPIENT_GROUP)
```

Comment:
Same question, do we need this edit breadcrumb?

Reply:
yes, we do other wise just the name breadcrumb doesn't appear, it comes like this if i remove the line `breadcrumbs.push(BREADCRUMBS.EDIT_RECIPIENT_GROUP)`
![Image 9-6-24 at 10 43 AM](https://github.com/user-attachments/assets/6344c873-b8f5-4b6c-819f-008034ead3e6)

Reply:
Aah, I see what you mean, ignore my comment.